src/__tests__/standalone-syntax-test.js

```diff
@@ -133,3 +133,87 @@ test("standalone with syntax set by extension", t => {
 
   t.end()
 })
+
+test("standalone with path to custom parser", t => {
+  const config = {
+    rules: {
+      "block-no-empty": true,
+    },
+  }
+
+  standalone({
+    config,
+    syntax: `${fixturesPath}/custom-parser`,
+    code: ".foo { width: 200px; }\n.bar {}",
+    formatter: stringFormatter,
+  }).then(({ output }) => {
+    const strippedOutput = chalk.stripColor(output)
+    t.equal(typeof output, "string")
+    t.ok(strippedOutput.indexOf("2:6") !== -1)
+    t.ok(strippedOutput.indexOf("block-no-empty") !== -1)
```

Comment:
What about this output assures us that the custom parser was used? How would this test fail if stylelint *did not* use the custom parser?